vendor-bin/php-cs-fixer/composer.json

```
@@ -0,0 +1,9 @@
+{
+	"require": {
+		"php": "^8.1 || ^8.2 || ^8.3 || ^8.4"
```

```suggestion
		"php": "^8.2 || ^8.3 || ^8.4"
```
since 8.1 was dropped